examples/tutorials/converter/xv15_bet_line_hover_good_input.json

```diff
@@ -0,0 +1,378 @@
+                rotation_direction_rule="leftHand",
+                number_of_blades=3,
+                omega=0.0046,
+                chord_ref=14,
+                n_loading_nodes=20,
+                blade_line_chord=25,
+                initial_blade_direction=[
```

Comment:
it looks like invalid JSON. All keys should be ""

Reply:
it is formatted in a way that allows for direct pasting into the new SimulationParams python code